lib/videojs-playlist-ui.vertical.less

```diff
@@ -0,0 +1,139 @@
+/* The default color for the playlist menu background, almost black */
+@background-color: #1a1a1a;
+/* The color used to emphasize the currently playing video and for
+hover effects */
+@highlight-color: rgba(26, 26, 26, 0.5);
+/* The primary foreground color */
+@main-color: #fff;
+/* The alternate foreground color */
+@secondary-color: #a6a6a6;
+
+@play-icon: '\e001';
+
+/* Rules common to mouse and touch devices */
+.vjs-playlist {
+  list-style-type: none;
+  overflow: auto;
+
+  .vjs-playlist-item {
+    cursor: pointer;
+    overflow: hidden;
+    position: relative;
+  }
+
+  .vjs-playlist-thumbnail {
+    background-color: @secondary-color;
+    display: block;
+    float: left;
+  }
+  .vjs-playlist-thumbnail::before {
+    display: block;
+    position: absolute;
+    background-color: @highlight-color;
+    text-align: center;
+  }
+  .vjs-selected .vjs-playlist-thumbnail::before {
+    content: 'Now Playing';
+    top: 0;
+    left: 0;
+  }
+  img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .vjs-playlist-duration {
+    background-color: rgba(26, 26, 26, 0.8);
+    left: 0;
+    padding: 2px 8px;
+    position: absolute;
+    top: 0;
+  }
+  .vjs-selected .vjs-playlist-duration {
+    display: none;
+  }
+
+  cite {
```

Comment:
we probably should add a class to this element and reference it that way instead.